modules/ACLRoles/ACLRole.php

```diff
@@ -56,25 +56,6 @@ class ACLRole extends SugarBean
 
     public $created_by;
 
-    public function __construct()
```

Comment:
Not big deal removing this constructor as it only calls the parent constructor, which is the default anyways. However it does not follow what it has been done in the rest of the patch.